test/server/users.js

```diff
@@ -159,15 +159,16 @@ describe('Users features', function () {
 				});
 				it(`should allow 'u' permissions on lower ranked users`, function () {
 					const user = new User();
-					user.tempGroup = '@';
+					user.forceRename("Moderator");
+					user.setGroup('@');
```

Comment:
Why `setGroup`? Did you change how `tempGroup` works?

Reply:
No, I just changed that in my attempts to figure out what was wrong and then never changed it back, since it sets tempGroup and i made User.can work with tempGroup.

Reply:
Can you change it back now, then?

Reply:
Alright.